packages/CollabCodeEdit-Client.package/CCEModel.class/instance/CCEackHeartbeat..st

```diff
@@ -0,0 +1,4 @@
+requests backend
+CCEackHeartbeat: anOrderedDictionary
+	" this has to be a method, to be able to correctly decode the ackHeartbeat. However it can be empty til we need the heartbeat to do more than just keeping our connection alive"
+	
```

Comment:
I know the review is NOT for commenting on small style errors, but I do it anyway. Empty Line here ...

Reply:
don't you dare!

Reply:
To be honest. A review is exactly for that. But after a comment an empty line is mandotory.

Reply:
https://blog.juliobiason.net/thoughts/things-i-learnt-the-hard-way/?utm_source=hackernewsletter&utm_medium=email&utm_term=fav ... Code reviews are not for style.

Reply:
Well we should discuss on that. In my opinion escpecially for SWT review should be for style.